packages/ngtools/webpack/src/ngcc_processor.ts

```diff
@@ -67,6 +67,17 @@ export class NgccProcessor {
       return;
     }
 
+    // If the package.json is read only we should skip calling NGCC.
+    // Under Bazel when running under sandbox the filesystem is read-only.
+    try {
```

Comment:
Thoughts... should we do the below for each module? Or if `node_modules` directory is read-only we don't invoke this processor from in the first place?

Reply:
What about `dist` type folders?

Reply:
Good point!